frontend/src/views/camp/CampProgram.vue

```diff
@@ -24,16 +24,16 @@ Show all activity schedule entries of a single period.
         <v-skeleton-loader v-else type="table" />
       </template>
       <template v-if="firstPeriod">
-        <activity-list
-          v-if="listFormat"
-          :camp="camp" :schedule-entries="firstPeriod.scheduleEntries().items" />
         <picasso
-          v-else
-          :camp="camp"
+          v-show="!listFormat"
```

Comment:
https://www.digitalocean.com/community/tutorials/vuejs-conditional-directives#v-show
> Usually, v-show has a performance advantage if the elements are switched on and off frequently, while the v-if has the advantage when it comes to initial render time.

Warum nicht mehr `v-if`? Und falls du doch wieder `v-if` verwendest, könntest du die Änderungen in diesem File komplett reverten.